go.mod

```diff
@@ -232,7 +232,7 @@ require (
 	k8s.io/component-base v0.28.1
 	k8s.io/klog/v2 v2.100.1
 	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
-	sigs.k8s.io/controller-runtime v0.16.2
+	sigs.k8s.io/controller-runtime v0.16.2 // TODO: Remove custom rest mapper from component/prometheus/operator/common/crdmanager.go when upgrading past v0.17.0
```

Comment:
Is the comment meant to be in the `go.mod`? 👀

Reply:
Yes, I intentionally put it here so that it will be visible in the diff when someone does upgrade this dependency

Reply:
Cool, thanks for clarifying!